frontend/src/components/molecules/FilterComponent/index.tsx

```diff
@@ -34,7 +34,6 @@ const FilterComponent = ({
         type="text"
         value={(value ?? '') as string}
         onChange={inputOnChange}
-        icon="filter"
```

Comment:
we should remove the icon props in the Input component since it's not used anymore.